code/src/components/Form.js

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react'
+
+import { UserName } from './Name'
+import { AgeRadioOption } from './Age'
+import { SelectPlanet } from './SelectPlanet'
+import { BringItem } from './BringItem'
+import { Summary } from './Summery'
+
+
+
+
+export const Form = () => {
+  const [ageGroupSpan, setAgeGroup] = useState('')
+  const [name, setName] = useState('')
+  const [item, setInputItem] = useState('')
+  const [planet, setPlanet] = useState('')
+  const [summary, setSummary] = useState(false)
+
+  const handelSubmit = event => {
+    event.preventDefault()
+    setSummary(true)
+  }
+
+ return (
```

Comment:
As a general comment regarding the indentation of the code here in the return section, I would suggest you to use an auto formatter to get the correct indentation (2 spaces according to the guidelines). Good indentation is great for readability 🙂